docker-compose.yml

```diff
@@ -7,8 +7,12 @@ services:
     networks:
       - back-tier
   web:
-    image: markstory/lint-review
-    command: 'gunicorn -c /code/settings.py lintreview.web:app'
+    build: .
```

Comment:
I don't really know :/
Personally I run my own copy of docker-compose.yml, and that pulls from hub.docker.com. So if this one is intended to be local-dev, then building is fine.